<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QWidget" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>configurer apitux</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>ligne :
(la 1° ligne = 0).</string>
       </property>
       <property name="buddy">
        <cstring>indexEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>mot choisi:</string>
       </property>
       <property name="buddy">
        <cstring>choosenwordEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Titre choisi :</string>
       </property>
       <property name="buddy">
        <cstring>titleEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="indexEdit">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="choosenwordEdit">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="titleEdit">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="titlesComboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Ajouter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Enregistrer le titre</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>E&amp;ffacer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Pour configurer l'intégration dans les Documents de MedinTux avec les bons titres, vous écivez le numéro de ligne choisie dans le document type dans la première case, le mot clé choisi dans la ligne dans la deuxième case et le titre dans la troisième case, par exemple BIO, CARDIO, CRH, CRO, ANAPATH, etc...&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Attention : ne pas inscrire de signe = dans le titre.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="utiltab">
      <attribute name="title">
       <string>Utilisateur</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>361</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Configurer l'utilisateur préférentiel :</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>324</width>
         <height>53</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>Choisir
cet utilisateur</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="purgetab">
      <attribute name="title">
       <string>Purge</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>340</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Purger la réserve
des fichiers sauvegardés</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
